plugins/inventory/linode.py

```diff
@@ -203,9 +219,27 @@ def parse(self, inventory, loader, path, cache=True):
 
         self._get_instances_inventory()
 
+        self._strict = self.get_option('strict')
```

Comment:
How about using a local variable here?

Reply:
Yeah, Originally I tried to put those calls in related private methods, but the ordering did not work out, so they all got moved to parse, so an instance variable is no longer needed.  The change has been pushed.